packages/firestore/src/api/database.ts

```diff
@@ -612,37 +618,14 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
     return new WriteBatch(this);
   }
 
-  static get logLevel(): firestore.LogLevel {
-    switch (getLogLevel()) {
-      case LogLevel.DEBUG:
-        return 'debug';
-      case LogLevel.SILENT:
-        return 'silent';
-      default:
-        // The default log level is error
-        return 'error';
-    }
+  static get logLevel(): LogLevel {
```

Comment:
Can you change this back to `firestore.LogLevel`, remove the use of the `LogLevel` type (we should only use LogLevelString) and change the typings for `firestore.LogLevel` to match the log types in Util?